recipes/neutronics-material-maker/meta.yaml

```
@@ -0,0 +1,47 @@
+
+{% set name = "neutronics_material_maker" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 841959e2af03213c135dc961ba90d5a0187ede7d72e0cbfd8ecf64242281c36d
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<35]
```

This could instead be noarch python which would simplify building things:
```suggestion
  noarch: python
```

The Python version requirement could then be added below in both `run` and `host` as `python >=3.6` (python 3.5 is no longer built on conda-forge)